### PR TITLE
Pipeline task event display optimization

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:
task events optimization
get pod inspect, only reveal task events when task scheduler failed

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/dop/projects/387/issues/all?id=204653&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMTIwNSJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=-1&type=BUG)

### CODE REVIEW

File: modules/pipeline/services/pipelinesvc/detail.go [hunk 1]
```diff
@@ -476,3 +477,10 @@ func findRunningStageID(p spec.Pipeline, tasks []spec.PipelineTask) uint64 {
 	}
 	return runningStageID
 }
+
+// isSuccessTaskEvents if events contain Successfully assigned like
+// "Successfully assigned project-387-dev/pipeline-3feb156fc4-65d7476d75-7m6qm to node-010000006200"
+// means scheduler successfully execute task
+func isSuccessTaskEvents(events string) bool {
+	return strings.Contains(events, "Successfully assigned")
```
Comment (sfwn) [inline modules/pipeline/services/pipelinesvc/detail.go:485]: all events don't contain `Successfully assigned` are failed?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/pipeline/services/pipelinesvc/detail.go (hunk 1 above)
Comment (chengjoey) [reply modules/pipeline/services/pipelinesvc/detail.go:485]: if task it`s not success and task don`t asigned, should reveal the events

**Comment by ZLValien** [comment]
Please indicate in the title what is fixed instead of what code is modified， please modify the title

**Comment by sfwn** [comment]
rebase master and push again to trigger CODE-CHECK.

**Pull request merged**